products/extensions/subscriptions/guides/mixed-checkout.md

```diff
@@ -0,0 +1,164 @@
+---
+nav:
+  title: Mixed checkout
+  position: 20
+
+---
+# Mixed subscription checkout
+
+::: info
+Available since Shopware version 6.7.4.0
+:::
+
+This guide describes how the mixed carts in the subscription feature works and how extensions should integrate with it.
+Mixed carts let customers buy subscription products and one‑time products in the same checkout while keeping subscription calculation isolated and predictable.
+
+Please familiarise yourself with the [concept](../concept.md) first before continuing here.
+
+## Overview
+
+Subscription line items are ordinary product line items in the main cart, but they carry subscription plan and interval IDs in their payload.
+During cart calculation, line items containing subscription plan and interval IDs in their payload are collected and grouped by plan and interval.
+For each group a derived _managed_ subscription context and a _managed_ subscription cart are created and calculated using the subscription cart calculation path.
+These represent the context and content of the later generated subscription.
+
+Managed subscription contexts and carts are persisted in the database as well.
+They are linked back to the main context by the `subscription_cart` database table.
+
+## Retrieving information
+
+You can access the managed carts through the cart extension named `subscriptionManagedCarts`, which maps keys in the form `<plan-id>-<interval-id>` to their corresponding [managed cart](../concept.md#subscription-cart).
+The sales channel context extension named `subscriptionManagedContexts` provides the same mapping for [managed sales channel contexts](../concept.md#subscription-context).
+The mapping is considered stable and the plan and interval IDs can be retrieved by splitting the composite ID.
```

Comment:
What do you mean with `stable` ?
It could be misleading, as `stable` usually means, that the structure is considered productive. I think you want to say sth else here?

Reply:
I would like to say that this composite ID will not change, in terms of either the order or the fact that plan and interval IDs are used. Splitting the ID is the main way of obtaining both IDs, so there is no need to worry that this process will not be successful in the future